### PR TITLE
Update draft g9 questions

### PR DESCRIPTION
**Pull request by samuelhwilliams**
* Update draft g9 questions for software, hosting, and support.
* Bump version to 5.14.1

### CODE REVIEW

File: frameworks/g-cloud-9/questions/services/multiqCustomisation.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+depends:
+  - "on": lot
+    being:
+      - cloud-software
+
+type: multiquestion
+questions:
+  - customisationAvailable
+  - whatCanBeCustomised
```
Comment (TheDoubleK) [inline frameworks/g-cloud-9/questions/services/multiqCustomisation.yml]: `customisationDetails` or `customisationDescription`?

File: frameworks/g-cloud-9/questions/services/commandLineOS.yml [hunk 2]
```diff
@@ -1,17 +1,18 @@
...
 options:
-    -
-        label: "Linux"
-    -
-        label: "Windows"
-    -
-        label: "Mac"
+  - label: Linux or Unix
+  - label: Windows
+  - label: Mac OS
```
Comment (ralph-hawkins) [inline frameworks/g-cloud-9/questions/services/commandLineOS.yml]: I've updated in question doc.

File: frameworks/g-cloud-9/questions/services/dataProtectionWithinSuppliersNetwork.yml [hunk 3]
```diff
@@ -1,20 +1,20 @@
...
-        label: "TLS (Version 1.2 or above)"
-    -
-        label: "IPsec or TLS VPN gateway"
-    -
-        label: "Legacy SSL and TLS (under 1.2)"
+  - label: Private WAN service
+  - label: TLS (Version 1.2 or above)
+  - label: IPsec or TLS VPN gateway
+  - label: Legacy SSL and TLS (under 1.2)
+  - label: Other
```
Comment (ralph-hawkins) [inline frameworks/g-cloud-9/questions/services/dataProtectionWithinSuppliersNetwork.yml]: Yeah. We have 'Describe how else you protect data between the buyer’s network and your network.' as a follow up to users choosing other. Otherwise we would have to make the question optional (remember our discussion about checkboxes behaving like buttons). We could wrap it in 'Do you protect data within your network?'.

File: frameworks/g-cloud-9/questions/services/dataSanitisationTypeSoftware.yml [hunk 4]
```diff
@@ -0,0 +1,18 @@
+id: dataSanitisationType
+name: Data sanitisation type
+question: Do you have a data sanitisation process?
+
+depends:
+  - "on": lot
+    being:
+      - cloud-software
+
+type: radios
```
Comment (ralph-hawkins) [inline frameworks/g-cloud-9/questions/services/dataSanitisationTypeSoftware.yml]: Yeah, have wrapped it in 'Do you have a data sanitisation process?', made it checkbox and removed the 'No data sanitisation process' option. We could reword it to be something about primary form of data sanitisation but would be a bit open to interpretation.

File: frameworks/g-cloud-9/questions/services/describeManagementAuthentication.yml [hunk 5]
```diff
@@ -0,0 +1,19 @@
+name: Describe management authentication
+question: Describe how you authentication management access to your service.
```
Comment (ralph-hawkins) [inline frameworks/g-cloud-9/questions/services/describeManagementAuthentication.yml]: Looks like a typo 'How do you authenticate management access to your service?'

File: frameworks/g-cloud-9/questions/services/devicesUsersManageTheServiceThrough.yml [hunk 6]
```diff
@@ -1,15 +1,15 @@
 name: Devices users manage the service through
 question: Through which devices do users manage the service?
```
Comment (ralph-hawkins) [inline frameworks/g-cloud-9/questions/services/devicesUsersManageTheServiceThrough.yml]: Made the suggestion in question doc.

File: frameworks/g-cloud-9/questions/services/howLongBuyersAuditDataStored.yml [hunk 7]
```diff
@@ -0,0 +1,20 @@
+name: How long buyers’ users’ audit data is stored
```
Comment (ralph-hawkins) [inline frameworks/g-cloud-9/questions/services/howLongBuyersAuditDataStored.yml]: Added comment to question doc.

File: frameworks/g-cloud-9/questions/services/dataSanitisationTypeSoftware.yml [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+options:
+  - label: Explicit overwriting of storage before reallocation
+  - label: Assurances media can’t be directly ’addressed’
```
Comment (ralph-hawkins) [inline frameworks/g-cloud-9/questions/services/dataSanitisationTypeSoftware.yml]: Added comment to question doc.

File: frameworks/g-cloud-9/manifests/edit_submission.yml [hunk 9]
```diff
@@ -19,70 +19,73 @@
...
     - serviceConstraints
-    - buyerResponsibilities
+    - systemRequirements
 -
   name: Reselling
   editable: false
   edit_questions: true
   questions:
-    - reselling
+    - supplierType
```
Comment (samuelhwilliams) [inline frameworks/g-cloud-9/manifests/edit_submission.yml]: I've tried to keep the filenames the same as they name/headings of the questions so it's easier to find the file from looking at the content on the page. This feels more valuable to me but happy to discuss if you think otherwise.

File: frameworks/g-cloud-9/questions/services/boardLevelServiceSecurity.yml [hunk 10]
```diff
@@ -1,5 +1,8 @@
-name: "Named, board-level person responsible for service security"
-question: "Does your organisation have a named person with board-level (or equivalent) authorisation who's responsible for the security of all of your services?"
+name: Named board-level person responsible for service security
+question: >
+  Does your organisation have a named person with board-level (or equivalent) authorisation who’s responsible for the
+  security of all of your services?
+
 depends:
   -
     "on": lot
```
Comment (samuelhwilliams) [inline frameworks/g-cloud-9/questions/services/boardLevelServiceSecurity.yml]: Nice spot, thanks.

File: frameworks/g-cloud-9/questions/services/serviceDescription.yml [hunk 11]
```diff
@@ -1,21 +1,23 @@
...
       - cloud-hosting
       - cloud-software
       - cloud-support
+
 type: textbox_large
 max_length_in_words: 50
+
 validations:
   -
     name: answer_required
```
Comment (samuelhwilliams) [inline frameworks/g-cloud-9/questions/services/serviceDescription.yml]: Nice spot, thanks.

File: frameworks/g-cloud-9/questions/services/commandLineOS.yml (hunk 2 above)
Comment (samuelhwilliams) [inline frameworks/g-cloud-9/questions/services/commandLineOS.yml]: Updated.

File: frameworks/g-cloud-9/questions/services/compatibleOperatingSystems.yml [hunk 12]
```diff
@@ -0,0 +1,21 @@
...
+depends:
+  - "on": lot
+    being:
+      - cloud-software
+
+type: checkboxes
+options:
+  - label: iOS
+  - label: Linux or Unix
+  - label: Mac OS
```
Comment (samuelhwilliams) [inline frameworks/g-cloud-9/questions/services/compatibleOperatingSystems.yml]: Updated.

File: frameworks/g-cloud-9/questions/services/describeManagementAuthentication.yml (hunk 5 above)
Comment (samuelhwilliams) [inline frameworks/g-cloud-9/questions/services/describeManagementAuthentication.yml]: Yes, typo, thanks.

File: frameworks/g-cloud-9/questions/services/multiqSecurityGovernance.yml [hunk 13]
```diff
@@ -0,0 +1,15 @@
...
+  - "on": lot
+    being:
+      - cloud-hosting
+      - cloud-software
+
+type: multiquestion
+questions:
+  - securityGovernanceAccreditation
+  - securityGovernanceStandards
+  - otherSecurityGovernanceStandards
```
Comment (samuelhwilliams) [inline frameworks/g-cloud-9/questions/services/multiqSecurityGovernance.yml]: Same as before, have tried to keep filenames the same as the question name/heading. This seems like a good argument against doing that, but maybe let's discuss...

File: frameworks/g-cloud-9/questions/services/schedulingBackups.yml [hunk 14]
```diff
@@ -1,20 +1,17 @@
...
-    -
-        label: "Users schedule backups through a web interface"
-        filter_label: "Web interface"
-    -
-        label: "Users contact the support team to schedule backups"
-        filter_label: "Support team"
-    -
-        label: "Supplier controls the whole backup schedule"
-        filter_label: "Supplier-controlled"
+  - label: Users schedule backups through a web interface
```
Comment (samuelhwilliams) [inline frameworks/g-cloud-9/questions/services/schedulingBackups.yml]: They're generally ordered from 'best'/'most preferable' to 'worst'/'least preferable'; if there's no clear way to order them like this, it falls back to alphabetical.

File: frameworks/g-cloud-9/questions/services/sfiaRateDocumentURL.yml [hunk 15]
```diff
@@ -0,0 +1,20 @@
+name: Skills Framework for the Information Age rate card
+question: Skills Framework for the Information Age (SFIA) rate card
+hint: Please upload your SFIA rate card if you have one. This document will not be indexed by search on the Digital Marketplace. Use an Open Document Format (ODF) or PDF/A (eg .pdf, .odt). (Maximum file size 5MB.)
+
+optional: yes
```
Comment (samuelhwilliams) [inline frameworks/g-cloud-9/questions/services/sfiaRateDocumentURL.yml]: Yes, done.

File: frameworks/g-cloud-9/questions/services/serviceName.yml [hunk 16]
```diff
@@ -1,19 +1,19 @@
 id: serviceName
-question: 'What’s your service called?'
-name: 'Service name'
-hint: 'Include your service name only. Don’t use extra keywords.'
+question: What’s your service called?
+name: Service name
+hint: Include your service name only. Don’t use extra keywords.
```
Comment (samuelhwilliams) [inline frameworks/g-cloud-9/questions/services/serviceName.yml]: Agree, but it's a question for @superroz @ralph-hawkins

File: frameworks/g-cloud-9/questions/services/serviceCategoriesHosting.yml [hunk 17]
```diff
@@ -1,21 +1,19 @@
...
+  - "on": lot
     being:
       - cloud-hosting
+
 type: checkboxes
 options:
-  -
-    label: Hosting type 1
+  - label: Hosting type 1
     value: hosting_category_one
```
Comment (samuelhwilliams) [inline frameworks/g-cloud-9/questions/services/serviceCategoriesHosting.yml]: Done.

File: frameworks/g-cloud-9/questions/services/multiqWebChatSupport.yml [hunk 18]
```diff
@@ -0,0 +1,16 @@
...
+question: Web chat support
+
+depends:
+  - "on": lot
+    being:
+      - cloud-hosting
+      - cloud-software
+
+type: multiquestion
+questions:
```
Comment (samuelhwilliams) [inline frameworks/g-cloud-9/questions/services/multiqWebChatSupport.yml]: 👍

File: frameworks/g-cloud-9/questions/services/multiqUserAuthenticationSoftware.yml [hunk 19]
```diff
@@ -0,0 +1,13 @@
...
+
+depends:
+  - "on": lot
+    being:
+      - cloud-software
+
+type: multiquestion
+questions:
+  - userAuthenticationNeeded
+  - userAuthentication
```
Comment (samuelhwilliams) [inline frameworks/g-cloud-9/questions/services/multiqUserAuthenticationSoftware.yml]: Not sure if it works, but interested to find out! Feels like it /should/, but could be wrong.

File: frameworks/g-cloud-9/questions/services/multiqScalingSoftware.yml [hunk 20]
```diff
@@ -0,0 +1,13 @@
...
+question: Scaling
+
+depends:
+  - "on": lot
+    being:
+      - cloud-software
+
+type: multiquestion
+questions:
+  - scalability
```
Comment (samuelhwilliams) [inline frameworks/g-cloud-9/questions/services/multiqScalingSoftware.yml]: Done.

File: frameworks/g-cloud-9/questions/services/multiqScalingHosting.yml [hunk 21]
```diff
@@ -0,0 +1,14 @@
...
+question: Scaling
+
+depends:
+  - "on": lot
+    being:
+      - cloud-hosting
+
+type: multiquestion
+questions:
+  - serviceScales
```
Comment (samuelhwilliams) [inline frameworks/g-cloud-9/questions/services/multiqScalingHosting.yml]: Done.

File: frameworks/g-cloud-9/questions/services/configurationAndChangeManagementType.yml [hunk 22]
```diff
@@ -1,20 +1,17 @@
...
       - cloud-hosting
       - cloud-software
 
 type: radios
 options:
-    -
-        label: "Conforms to a recognised standard, for example CSA CCM v3.0 or SSAE-16 / ISAE 3402"
-    -
-        label: "Supplier-defined controls"
+  - label: Conforms to a recognised standard, for example CSA CCM v3.0 or SSAE-16 / ISAE 3402
```
Comment (samuelhwilliams) [inline frameworks/g-cloud-9/questions/services/configurationAndChangeManagementType.yml]: Values added, thanks.

File: frameworks/g-cloud-9/manifests/edit_submission.yml [hunk 23]
```diff
@@ -19,70 +19,73 @@
...
-    - emailSupport
-    - onlineTicketingSupport
-    - onlineTIcketingSupportAccessibility
-    - supportResponseTimes
-    - phoneSupport
-    - phoneSupportAvailability
-    - webChatSupport
-    - webChatSupportAvailability
-    - webChatSupportAccessibility
+    - multiqEmailOrTicketingSupport
```
Comment (samuelhwilliams) [inline frameworks/g-cloud-9/manifests/edit_submission.yml]: 👍

File: frameworks/g-cloud-9/manifests/edit_submission.yml (hunk 9 above)
Comment (samuelhwilliams) [inline frameworks/g-cloud-9/manifests/edit_submission.yml]: Nevermind, thinking again I like your idea more. I'll go through and fix up.

File: frameworks/g-cloud-9/questions/services/freeVersionLink.yml [hunk 24]
```diff
@@ -0,0 +1,18 @@
+name: Free version link
```
Comment (samuelhwilliams) [inline frameworks/g-cloud-9/questions/services/freeVersionLink.yml]: Done.

File: frameworks/g-cloud-9/manifests/edit_submission.yml [hunk 25]
```diff
@@ -169,28 +164,28 @@
...
 
 -
   name: Identity and authentication
   editable: false
   edit_questions: true
   questions:
-    - userAuthentication
+    - multiqUserAuthenticationHosting
+    - multiqUserAuthenticationSoftware
     - accessRestrictionsManagementAndSupport
```
Comment (samuelhwilliams) [inline frameworks/g-cloud-9/manifests/edit_submission.yml]: Done.

**Comment by TheDoubleK** [comment]
Let's do this and tweak later.
I think the multiquestion keys thing is definitely a good thing.
 :+1:
